src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -500,6 +500,15 @@ export class BrightScriptDebugSession extends BaseDebugSession {
                     try {
                         let functionName = await this.fileManager.getCorrectFunctionNameCase(sourceLocation.filePath, debugFrame.functionIdentifier);
                         if (functionName) {
+
+                            //search for original function name if this is an anonymous function
+                            if (functionName.startsWith('$anon')) {
```

Comment:
The core of this PR. Anytime we find a function that starts with `$anon`, we try to look up an original function name. This mainly fixes class-transpiled-to-anonymous-function patterns.